eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -6,33 +6,23 @@ parameters:
   RepoId: 'not-specified'
```

Comment:
Looks like there a are a number of unused parameters.

Reply:
Just one. Fixed.